lang/cs/Org.Apache.REEF.Network/Naming/NameClient.cs

```diff
@@ -64,13 +65,33 @@ public class NameClient : INameClient
         /// <param name="remoteAddress">The ip address of the NameServer</param>
         /// <param name="remotePort">The port of the NameServer</param>
         [Inject]
-        public NameClient(
+        private NameClient(
```

Comment:
Did we ever release with this being `public`? If so, we need to `[Obsolete]` it first, then make it private in 0.13.